docs/custom-domain/README.md

```diff
@@ -1,105 +1,128 @@
-# Using Custom Domains with Azure Container Registry
+# Use a custom domain with Azure Container Registry (private preview)
 
-Every ACR is accessed using its login server. If you have a registry called `myregistry`, you access it using its default hostname, `myregistry.azurecr.io` (in Azure Public Cloud.) As a customer belonging to an organization, you may prefer to access your registry using a custom domain that is associated with your organization, for instance, `container-registry.contoso.com`.
+This article describes the steps to set up the endpoints of an Azure container registry using your own custom domain name, such as **contoso.com**.
 
-The following steps describe how you can achieve this.
+By default, every Azure container registry is accessed externally using its login server name. If you have a registry called `myregistry`, you access it using its default hostname, `myregistry.azurecr.io` (in the Azure Cloud). As a customer belonging to an organization, you may prefer to access your registry using a custom domain name that is associated with your organization, for example, `myregistry.contoso.com`.
+
+**Important - Using a custom domain name in Azure Container Registry is currently a private preview feature**. It can only be enabled by preparing certificates, a Premium container registry, and an Azure key vault as described in this article, and then opening an Azure support request to complete the configuration. 
+
+## Limitations
+
+* This capability is provided without a service level agreement, and isn't currently recommended for production workloads.
+* Azure Container Registry custom domains can't currently be used where key vault access is restricted. 
```

Comment:
Might want to elaborate on what "restricted means". Are we talking about auth, or private link?

Reply:
Updated to "Azure Container Registry custom domains can't currently be configured using an Azure key vault deployed in a virtual network with a Key Vault firewall."

Reply:
If a system assigned managed identity is used, key vault firewall can be bypassed if Trusted Service Bypass is enabled. This is not true of user-assigned managed identities though.

Reply:
@aviral26 - Removed the limitation and added information to "Enhanced security with virtual network"